.env.example

```diff
@@ -10,3 +10,4 @@ EVIDENCE_REQUEST_CLIENT_URL=http://localhost:3000
 DOCUMENTS_API_URL=http://localhost:5001/
 DOCUMENTS_API_POST_CLAIMS_TOKEN=api-post-claims-token
 DOCUMENTS_API_POST_DOCUMENTS_TOKEN=api-post-documents-token
+DOCUMENTS_API_GET_CLAIMS_TOKEN=api-get-claims-token
```

Comment:
Could put this above `DOCUMENTS_API_POST_CLAIMS_TOKEN` so the order matches other files like `serverless.yml`